messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/token/store/jpa/JpaTokenStore.java

```diff
@@ -410,20 +414,20 @@ private ConfigToken getConfig() {
             token = new TokenEntry(CONFIG_TOKEN_ID,
                                    CONFIG_SEGMENT,
                                    new ConfigToken(Collections.singletonMap("id", UUID.randomUUID().toString())),
-                                   serializer);
+                                   converter);
             em.persist(token);
             em.flush();
         }
-        return (ConfigToken) token.getToken(serializer);
+        return (ConfigToken) token.getToken(converter);
     }
 
     /**
      * Returns the serializer used by the Token Store to serialize tokens.
      *
      * @return the serializer used by the Token Store to serialize tokens
      */
-    public Serializer serializer() {
-        return serializer;
+    public Converter serializer() {
```

Comment:
Is this method used? If not, I'd prefer we don't have it at all. These getters on our infrastructure components are from a different era.